.github/workflows/lock-threads.yml

```diff
@@ -20,15 +20,16 @@ jobs:
     timeout-minutes: ${{  fromJSON(vars.TIMEOUT_MINUTES_SHORT) }}
     runs-on: ${{ vars.RUNNER_NORMAL }}
     steps:
-      - uses: actions/stale@v8
+      - uses: actions/stale@v9
+        name: Regular stale action
         with:
           repo-token: ${{ secrets.CLERK_COOKIE_PAT }}
           days-before-issue-stale: 30
           days-before-pr-stale: 50
           days-before-issue-close: 10
           days-before-pr-close: 10
           exempt-all-assignees: true
-          exempt-issue-labels: 'needs-triage,prioritized,feature-request'
+          exempt-issue-labels: 'needs-triage,prioritized,confirmed,needs-reproduction'
```

Comment:
We don't want the regular action to run on issues that are marked with `needs-reproduction`

Reply:
❓ did we drop the `feature-request` tag in favor of productlane?

Reply:
yes